actualbudget/docker-compose.yml

```
@@ -0,0 +1,16 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: actualbudget_web_1
+      APP_PORT: 5006
+
+  web:
+    image: actualbudget/actual-server:23.5.0@sha256:e2e6daa61c9049e2a7b637500f18a1a2c4ddee9af4ac4b1d90d1f2f194383b47
+    restart: on-failure
+    volumes:
+      - ${APP_DATA_DIR}/data/:/data
+    environment:
+      - PUID=1000
+      - PGID=1000
```

It looks like this image doesn't accept these environment variables, but we can probably run as 1000:1000 without issue by setting the user directive:

```suggestion
  web:
    image: actualbudget/actual-server:23.5.0@sha256:e2e6daa61c9049e2a7b637500f18a1a2c4ddee9af4ac4b1d90d1f2f194383b47
    user: 1000:1000
    restart: on-failure
    volumes:
      - ${APP_DATA_DIR}/data/:/data
```

---

Sure, committing your suggestion 😉